test/ciphers.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -eufx
+
+export OPENSSL_ENGINES=${PWD}/.libs
```

Comment:
There were some simplification merged for the test scripts in a recent PR.
Please look at the current examples https://github.com/tpm2-software/tpm2-tss-engine/blob/master/test/rsasign.sh